wrappers/python/tests/anoncreds/test_verify_proof.py

```diff
@@ -56,6 +56,7 @@
                 "sub_proof_index": 0, "raw": "Alex", "encoded": "1139481716457488690172217916278103335"
             }
         },
+        "revealed_attr_groups": {},
```

Comment:
As far as I understand `revealed_attr_groups` is optional, why we've added it here?